CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## Unreleased
+
+**Features**:
+
+- Implement a new logger hook ([#267](https://github.com/getsentry/sentry-native/pull/267))
+  This adds the new `sentry_options_set_logger` function, which can be used to
+  customize the sentry-internal logging, for example to integrate into an apps
+  own logging system, or to stream logs to a file.
+- New CMake options: `SENTRY_LINK_PTHREAD`, `SENTRY_BUILD_RUNTIMESTATIC` and `SENTRY_EXPORT_SYMBOLS` along with other CMake improvements.
+
+**Fixes**:
+
+- Avoid memory unsafety when loading session from disk ([#270](https://github.com/getsentry/sentry-native/pull/270))
+- Flush scope immediately when starting Crashpad Backend ([#272](https://github.com/getsentry/sentry-native/pull/272))
```

Comment:
Should we write what the symptom of the bug was and what this changes? "Flushing scope" seems a bit technical.